src/Sylius/Bundle/ShopBundle/Resources/config/services/twig/component.xml

```diff
@@ -89,5 +89,57 @@
                 template="@SyliusShop/product/common/original_price.html.twig"
             />
         </service>
+
+        <service
+            id="sylius_shop.twig.component.product_summary"
+            class="Sylius\Bundle\ShopBundle\Twig\Component\Product\ProductSummaryComponent"
+        >
+            <argument type="service" id="sylius.product_variant_resolver.default" />
+
+            <tag
+                name="sylius.live_component"
+                key="sylius_shop:product:show_summary"
+                template="@SyliusShop/product/show/page/info/summary.html.twig"
+            />
+        </service>
+
+        <service
+            id="sylius_shop.twig.component.product_show_price"
+            class="Sylius\Bundle\ShopBundle\Twig\Component\Product\ProductShowPriceComponent"
+        >
+            <argument type="service" id="sylius.calculator.product_variant_price" />
+            <argument type="service" id="sylius.templating.helper.format_money" />
+            <argument type="service" id="sylius.context.channel" />
+            <argument type="service" id="sylius.context.locale" />
+            <argument type="service" id="sylius.context.currency" />
+
+            <tag
+                name="sylius.live_component"
+                key="sylius_shop:product:show_price"
+                template="@SyliusShop/product/show/page/info/summary/prices/price.html.twig"
+            />
+        </service>
+
+        <service
+            id="sylius_shop.twig.component.product_add_to_cart_form"
+            class="Sylius\Bundle\ShopBundle\Twig\Component\Product\AddToCart\FormComponent"
+        >
+            <argument type="service" id="form.factory" />
+            <argument type="service" id="sylius.factory.order_item" />
+            <argument type="service" id="sylius.factory.add_to_cart_command" />
+            <argument type="service" id="sylius.context.cart" />
+            <argument type="service" id="sylius.order_item_quantity_modifier" />
+            <argument>Sylius\Bundle\CoreBundle\Form\Type\Order\AddToCartType</argument>
```

Comment:
Do we want to do in the same way as admin bundle? https://github.com/Sylius/Sylius/pull/16262